Corona-Warn-App/src/main/res/values/strings.xml

```diff
@@ -367,6 +367,25 @@
     <!-- YTXT: one time risk explanation dialog - pointing to the faq page for more information-->
     <string name="risk_details_explanation_dialog_faq_body">"For further information, please see our FAQ page."</string>
 
+
+    <!-- ####################################
+              Target de
+    ###################################### -->
+
+    <!-- XBUT: onboarding - forward and deny -->
+    <string name="target_de_text_subtitle">"Haben Sie sich in den letzten 14 Tagen außerhalb von Deutschland aufgehalten?"</string>
```

Comment:
please change the ids to `submission_target_subtitle`, also the string should be empty. as it's in the default strings, e.g. `<string name="target_de_text_subtitle" />`

Reply:
also relevant for all other strings added in this PR

Reply:
And the strings should be also added in the german strings xml with the text